ivy/functional/backends/numpy/activations.py

```diff
@@ -30,24 +30,20 @@ def gelu(x, approximate: Optional[bool] = True):
     return 0.5 * x * (1 + erf(x / np.sqrt(2)))
 
 
-def sigmoid(x: np.ndarray, out: Optional[np.ndarray] = None) -> np.ndarray:
-    return 1 / (1 + np.exp(-x, out=out))
+def sigmoid(x: np.ndarray) -> np.ndarray:
+    return (1 / (1 + np.exp(-x)).astype(x.dtype))
 
 
-def tanh(x: np.ndarray, out: Optional[np.ndarray] = None) -> np.ndarray:
-    return (np.exp(x, out=out) - np.exp(-x, out=out)) / (
-        np.exp(x, out=out) + np.exp(-x, out=out)
-    )
+def tanh(
+    x: np.ndarray,
+) -> np.ndarray:
+    return (np.exp(x) - np.exp(-x)) / (np.exp(x) + np.exp(-x))
 
 
-def softmax(
-    x: np.ndarray, axis: Optional[int] = None, out: Optional[np.ndarray] = None
-) -> np.ndarray:
-    exp_x = np.exp(x, out=out)
-    return exp_x / np.sum(exp_x, axis, keepdims=True, out=out)
+def softmax(x: np.ndarray, axis: Optional[int] = None) -> np.ndarray:
```

Comment:
why are we removing `out` arguments? Same goes for other functions.

Reply:
Let me know if it is necessary to handle it explicitly in the backend functions instead @djl11. Thanks!

Reply:
we should always handle it in the backend function if we can, and so we should leave the arguments here

Reply:
Sure ! Thanks.